apps/web/src/studio/components/workflows/node-view/WorkflowFloatingMenu.tsx

```diff
@@ -41,12 +41,17 @@ export const WorkflowFloatingMenu: FC<IWorkflowFloatingMenuProps> = ({ className
             tooltipLabel="Guide of how to add a Delay step for embedding in code"
             onClick={handleClick('delay')}
           />
+          <WorkflowFloatingMenuButton
+            Icon={IconOutlineBolt}
+            tooltipLabel="Guide of how to add a Custom step for embedding in code"
+            onClick={handleClick('custom')}
+          />
         </WorkflowFloatingMenuSection>
         <WorkflowFloatingMenuSection title="Channels">
           <WorkflowFloatingMenuButton
             Icon={IconOutlineNotifications}
             tooltipLabel="Guide of how to add an In-app step for embedding in code"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Are we just changing the term? No change needed for the "In-app" tooltip label?‏